package.json

```diff
@@ -30,22 +30,23 @@
     "lint": "eslint src"
   },
   "dependencies": {
-    "@babel/helper-module-imports": "7.0.0-beta.56",
+    "@babel/helper-module-imports": "7.0.0-rc.2",
     "rollup-pluginutils": "^2.3.0"
   },
   "peerDependencies": {
-    "@babel/core": "7 || ^7.0.0-beta"
+    "@babel/core": "7 || ^7.0.0-rc.2",
```

Comment:
For peerDep we just did https://github.com/babel/babel/blob/47e05d70f31c160ff9832038985e3f687c35489f/packages/babel-cli/package.json#L33 but that should work too?

Reply:
Hm, I'm not sure what exact versions are catched by this range - 7.0.0-0 . I've wanted to explicitly allow only latest rc (and newer) because of added `caller`. It seems to me that the clear warning what should be done if used with wrong version of babel is enough and we don't have to handle (even temporarily) older versions, especially given the fact that rollup-plugin-babel is not used as often as babel-loader (which handles older versions for now)

Reply:
Yes ^7.0.0-rc.2 should work here fine.